### PR TITLE
Knowledge check

### PR DESCRIPTION
**Pull request by WillNock**
Pull Request to merge Knowledge Check & Training Dashboard features into develop

### CODE REVIEW

**Comment by sriharrsha** [comment]
@WillNock Submit again without merge conflicts

**Comment by sriharrsha** [comment]
@WillNock  Props need to be validated. Check the CI/ build recommendations.

**Comment by sriharrsha** [comment]
@clarah2 @WillNock Can you clear the merge conflicts. I don't know which one is the latest change.

File: src/components/TransitionComponents/TransitionHelpCard.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ import TableCell from '@material-ui/core/TableCell/index';
 import Typography from '@material-ui/core/Typography/index';
 
 
-const styles = theme => ({
+const styles = () => ({
```
Comment (sriharrsha) [inline src/components/TransitionComponents/TransitionHelpCard.js]: Why is the theme removed? Do you think it is correct?

File: src/views/protected/TransitionViews/TransitionTimeHelp.tsx [hunk 2]
```diff
@@ -22,9 +23,9 @@ const styles: object = {
   paper: {
     position: "absolute",
     width: "67%",
-    backgroundColor: theme.palette.background.paper,
-    boxShadow: theme.shadows[5],
-    padding: theme.spacing.unit * 4,
+    //backgroundColor: theme.palette.background.paper,
```
Comment (sriharrsha) [inline src/views/protected/TransitionViews/TransitionTimeHelp.tsx]: You had to comment these because theme was removed

Comment (clarah2) [reply src/views/protected/TransitionViews/TransitionTimeHelp.tsx]: @sriharrsha @WillNock I think this was me who removed them...but I changed the backgroundColor to 'white' and changed padding to '1em' and got rid of the boxShadow to make this more consistent with how we've done the other styles. That's how it is in develop now

**Pull request merged**